.changeset/config.json

```diff
@@ -2,55 +2,7 @@
   "$schema": "https://unpkg.com/@changesets/config@1.3.0/schema.json",
   "changelog": "@changesets/cli/changelog",
   "commit": false,
-  "linked": [
-    [
-      "example-app",
-      "@backstage/backend-common",
-      "example-backend",
-      "@backstage/catalog-model",
-      "@backstage/cli-common",
-      "@backstage/cli",
-      "@backstage/config-loader",
-      "@backstage/config",
-      "@backstage/core-api",
-      "@backstage/core",
-      "@backstage/create-app",
-      "@backstage/dev-utils",
-      "docgen",
-      "e2e-test",
-      "storybook",
-      "@techdocs/cli",
-      "@backstage/test-utils-core",
-      "@backstage/test-utils",
-      "@backstage/theme",
-      "@backstage/plugin-api-docs",
-      "@backstage/plugin-app-backend",
-      "@backstage/plugin-auth-backend",
-      "@backstage/plugin-catalog-backend",
-      "@backstage/plugin-catalog",
-      "@backstage/plugin-circleci",
-      "@backstage/plugin-gcp-projects",
-      "@backstage/plugin-github-actions",
-      "@backstage/plugin-gitops-profiles",
-      "@backstage/plugin-graphiql",
-      "@backstage/plugin-graphql-backend",
-      "@backstage/plugin-jenkins",
-      "@backstage/plugin-lighthouse",
-      "@backstage/plugin-newrelic",
-      "@backstage/plugin-proxy-backend",
-      "@backstage/plugin-register-component",
-      "@backstage/plugin-rollbar-backend",
-      "@backstage/plugin-rollbar",
-      "@backstage/plugin-scaffolder-backend",
-      "@backstage/plugin-scaffolder",
-      "@backstage/plugin-sentry-backend",
-      "@backstage/plugin-sentry",
-      "@backstage/plugin-tech-radar",
-      "@backstage/plugin-techdocs-backend",
-      "@backstage/plugin-techdocs",
-      "@backstage/plugin-welcome"
-    ]
-  ],
+  "linked": [["@backstage/*"]],
```

Comment:
Perhaps a plain `*` will capture everything? Think it makes sense to include the private packages too

Reply:
Doen in https://github.com/spotify/backstage/pull/2829/commits/8bb8a4a8d6e94a2761141c6db50742f9083338c9